src/main/scala/nl/knaw/dans/deposit/Deposit.scala

```diff
@@ -178,6 +179,32 @@ class Deposit private(val baseDir: File,
     withDepositProperties(newProperties)
   }
 
+  def withoutDansDoiAction: Deposit = {
```

Comment:
please put these methods in the same order as all others: (1) getter, (2) setter (3) unsetter